functions-and-operators/bit-functions-and-operators.md

```diff
@@ -12,13 +12,291 @@ TiDB supports all of the [bit functions and operators](https://dev.mysql.com/doc
+
+Before the operation, the value is expanded to 64 bits.
+
+Take the binary number `1111000011110000` as an example. When expanded to 64 bits and inverted, it looks like this:
+
+```
+Original (16 bits):                                                                 1111000011110000
+Expanded and inverted (64 bits):    1111111111111111111111111111111111111111111111110000111100001111
+```
+
+In SQL, you can use the `~` operator as follows:
+
+```sql
+SELECT CONV(~ b'1111000011110000',10,2);
++------------------------------------------------------------------+
+| CONV(~ b'1111000011110000',10,2)                                 |
++------------------------------------------------------------------+
+| 1111111111111111111111111111111111111111111111110000111100001111 |
++------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+You can reverse the inversion by applying the `~` operator again to the result:
+
+```sql
+SELECT CONV(~ b'1111111111111111111111111111111111111111111111110000111100001111',10,2);
+```
+
+```
++----------------------------------------------------------------------------------+
+| CONV(~ b'1111111111111111111111111111111111111111111111110000111100001111',10,2) |
++----------------------------------------------------------------------------------+
+| 1111000011110000                                                                 |
++----------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [Bitwise OR: `|`](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_bitwise-or)
+
+The `|` operator performs a bitwise OR operation, which compares the corresponding bits of two numbers and returns a new number based on whether at least one of the corresponding bits is set as 1.
+
+For example, a bitwise OR operation between `1010` and `1100` returns `1110`, because among the first three bits of the two numbers, at least one of the corresponding bits is set as 1.
+
+```
+  1010
+| 1100
+  ----
+  1110
+```
+
+In SQL, you can use the `|` operator as follows:
+
+```sql
+SELECT CONV(b'1010' | b'1100',10,2);
+```
+
+```
++------------------------------+
+| CONV(b'1010' | b'1100',10,2) |
++------------------------------+
+| 1110                         |
++------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [Bitwise XOR: `^`](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_bitwise-xor)
+
+The `^` operator performs a bitwise XOR (exclusive OR) operation, which compares each bit of two numbers and returns a new number based on whether the corresponding bits are different.
+
+For example, a bitwise XOR operation between `1010` and `1100` returns `0110`, because the first bits of the two numbers are different and the third bits of the two numbers are different.
```

Comment:
```suggestion
For example, a bitwise XOR operation between `1010` and `1100` returns `0110`, because the second and the third bits of the two numbers are different.
```